third_party/optimized-svg-icons/amp-social-share-svgs.css

```diff
@@ -5,45 +5,41 @@
 }
 
 .amp-social-share-line {
-  background-image: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 511.99"><defs><style>.cls-1{fill:#fff;}</style></defs><title>名称未設定-1</title><g id="レイヤー_1" data-name="レイヤー 1"><path class="cls-1" d="M443.2,233.29c0-84.14-84.35-152.6-188-152.6s-188,68.46-188,152.6c0,75.43,66.9,138.61,157.26,150.55,6.13,1.32,14.46,4,16.57,9.27,1.89,4.76,1.24,12.2.61,17,0,0-2.21,13.26-2.69,16.09-.82,4.75-3.78,18.6,16.29,10.14S363.45,372.58,402.9,327.18h0C430.15,297.29,443.2,267,443.2,233.29ZM188.82,278.21a3.67,3.67,0,0,1-3.66,3.67H132.47a3.6,3.6,0,0,1-2.53-1l-.06-.05,0-.05a3.65,3.65,0,0,1-1-2.53h0V196.29a3.66,3.66,0,0,1,3.66-3.66h13.19a3.66,3.66,0,0,1,3.66,3.66v65.07h35.84a3.66,3.66,0,0,1,3.66,3.66Zm31.8,0a3.65,3.65,0,0,1-3.66,3.65h-13.2a3.65,3.65,0,0,1-3.66-3.65V196.29a3.66,3.66,0,0,1,3.66-3.66H217a3.66,3.66,0,0,1,3.66,3.66Zm90.78,0a3.65,3.65,0,0,1-3.66,3.65H294.55a3.67,3.67,0,0,1-.94-.12h-.05l-.25-.08-.11,0-.18-.08-.17-.08-.11-.06-.22-.14,0,0a3.45,3.45,0,0,1-.93-.9L254,229.56v48.66a3.66,3.66,0,0,1-3.67,3.65H237.1a3.65,3.65,0,0,1-3.66-3.65V196.29a3.66,3.66,0,0,1,3.66-3.66h13.51l.19,0,.16,0,.21.05.13,0,.21.07.12,0a1.31,1.31,0,0,1,.21.08l.12.06.19.11a.41.41,0,0,1,.11.07l.19.13.1.07.19.16.07.07a2.28,2.28,0,0,1,.22.22l0,0a3.58,3.58,0,0,1,.28.37L290.89,245V196.29a3.66,3.66,0,0,1,3.66-3.66h13.19a3.66,3.66,0,0,1,3.66,3.66Zm72.83-68.74a3.66,3.66,0,0,1-3.65,3.67H344.74V227h35.84a3.66,3.66,0,0,1,3.65,3.67v13.19a3.65,3.65,0,0,1-3.65,3.66H344.74v13.85h35.84a3.65,3.65,0,0,1,3.65,3.66v13.19a3.66,3.66,0,0,1-3.65,3.67h-52.7a3.66,3.66,0,0,1-2.53-1l-.05-.05a.12.12,0,0,1-.05-.05,3.65,3.65,0,0,1-1-2.53h0V196.3h0a3.6,3.6,0,0,1,1-2.52l.06-.07s0,0,0,0a3.63,3.63,0,0,1,2.54-1h52.7a3.66,3.66,0,0,1,3.65,3.67Z"/></g></svg>');
+  background-image: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path fill="#fff" d="M443 233c0-84-84-152-188-152S67 149 67 233c0 76 67 139 157 151 7 1 15 4 17 9s1 12 1 17l-3 16c-1 5-4 19 16 10s108-63 148-109c27-30 40-60 40-94Zm-254 45a4 4 0 0 1-4 4h-53a4 4 0 0 1-2-1 4 4 0 0 1-1-3v-82a4 4 0 0 1 4-3h13a4 4 0 0 1 3 3v65h36a4 4 0 0 1 4 4Zm32 0a4 4 0 0 1-4 4h-13a4 4 0 0 1-4-4v-82a4 4 0 0 1 4-3h13a4 4 0 0 1 4 3Zm90 0a4 4 0 0 1-3 4h-13a4 4 0 0 1-1 0h-1v-1a3 3 0 0 1-1-1l-38-50v48a4 4 0 0 1-4 4h-13a4 4 0 0 1-4-4v-82a4 4 0 0 1 4-3h16v1l38 51v-49a4 4 0 0 1 4-3h13a4 4 0 0 1 3 3Zm73-69a4 4 0 0 1-3 4h-36v14h36a4 4 0 0 1 3 4v13a4 4 0 0 1-3 4h-36v13h36a4 4 0 0 1 3 4v13a4 4 0 0 1-3 4h-53a4 4 0 0 1-3-1 4 4 0 0 1-1-3v-82a4 4 0 0 1 1-2 4 4 0 0 1 3-1h53a4 4 0 0 1 3 3Z" data-name="レイヤー 1"/></svg>');
```

Comment:
These should all have `;charset=utf-8` directly after `svg+xml`

Reply:
Sounds good, especially since they have the title in non-ascii on the Line icon.
As a counter argument, I've seen most icons have only ASCII chars, which is the default encoding, so we could remove the `utf8` for those cases since the default value is `US-ASCII`. WDYT?

Reply:
There's an issue with some browsers when the chraset is missing, even if the contents are entirely ASCII. Essentially treat everything on the web as UTF8, even when it's not.

Reply:
@mszylkowski we should fix this asap the bug is in beta

Reply:
This should be fixed by https://github.com/ampproject/amphtml/pull/36987 and https://github.com/ampproject/amphtml/pull/37017.